ir/constant/const_float.go

```diff
@@ -105,7 +105,7 @@ func NewFloatFromString(typ *types.FloatType, s string) (*Float, error) {
 			// From https://llvm.org/docs/LangRef.html#simple-constants
 			//
 			// > The 128-bit format used by PowerPC (two adjacent doubles) is represented by 0xM followed by 32 hexadecimal digits.
-			hex := s[len("0xM"):] // first remove the prefix
+			hex := strings.TrimPrefix(s, "0xM") // first remove the prefix
```

Comment:
also remove these comments, as they are no longer needed.

Reply:
Just removed, and I also found line 197-198 is unreachable now, it's about unimplemented. I would also remove it